spec/models/like_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Like, type: :model do
+  let(:user1) { User.create(name: 'Tom', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Teacher from Kenya.', posts_counter: 0) }
+  let(:post) do
+    Post.create(author: user1, title: 'Hello', text: 'This is my first post', likes_counter: 0, comments_counter: 0)
```

Comment:
Some specs are failing for Like model. Kindly see the screenshot below for reference
![like_error](https://user-images.githubusercontent.com/35504149/177796631-7645b596-09ae-4583-8b32-40ec0d349f96.png)
.